build/Targets/Packages.props

```diff
@@ -255,6 +255,10 @@
     <xunitrunnervisualstudioVersion>2.3.1</xunitrunnervisualstudioVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <UsingToolPdbConverter>true</UsingToolPdbConverter>
```

Comment:
Why isn't this in Settings.props?

Reply:
I'm morphing this file to become `Versions.props`, which Arcade expects to find specification of dependencies in (versions, feeds, UsingToolXxx properties). One step at a time. It might be a bit odd now, but it will make sense at the end :)